session/manager_fake.go

```diff
@@ -1,9 +1,16 @@
 package session
 
+import "github.com/mysterium/node/identity"
+
+//ManagerFake represents fake manager usually useful in tests
 type ManagerFake struct{}
 
-func (manager *ManagerFake) Create() (Session, error) {
-	return Session{"new-id", "new-config"}, nil
+//Create function creates and returns fake session
```

Comment:
Most of comments for public objects seem to include space after `//`, that is `// Create...` - this seems more readable.

Reply:
This is also in all effective-go examples: https://golang.org/doc/effective_go.html#commentary

Reply:
Fixing wherever I see it.